godotopenxrkhr/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+    id 'com.android.library'
+}
+
+android {
+    compileSdk versions.compileSdk
+
+    defaultConfig {
+        minSdk versions.minSdk
+        targetSdk versions.targetSdk
+        versionCode versions.versionCode
+        versionName versions.versionName
+
+        ndk {
+            abiFilters 'arm64-v8a', 'armeabi-v7a',  'x86',  'x86_64'
+        }
+    }
+
+    namespace = "org.godotengine.openxrloaders.godotopenxrkhr"
```

Comment:
To match the format used for the other loaders, here and in the rest of the PR, the namespace should be `org.godotengine.openxrloaders.khr`

Reply:
Done!